util/resolver_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2017 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"testing"
+
+	"google.golang.org/grpc/naming"
+
+	"github.com/kylelemons/godebug/pretty"
+)
+
+func TestFixedBackendsResolver(t *testing.T) {
+	var tests = []struct {
+		target  string
+		wantErr bool
+		want    FixedBackends
+	}{
+		{target: "", wantErr: true},
+		{target: "a,b,c", want: FixedBackends{pendingServers: []string{"a", "b", "c"}}},
+		{target: "alongnamewithaport:111", want: FixedBackends{pendingServers: []string{"alongnamewithaport:111"}}},
+	}
+	for _, test := range tests {
+		fbr := FixedBackendResolver{}
+		got, err := fbr.Resolve(test.target)
+		if gotErr := (err != nil); gotErr != test.wantErr {
+			t.Errorf("FixedBackendResolver.Resolve(%q)=(%v,%v); want (_,err=%v)", test.target, got, err, test.wantErr)
+		}
+		if err != nil {
+			continue
+		}
+		if diff := pretty.Compare(got, test.want); diff != "" {
+			t.Errorf("FixedBackendResolver.Resolve(%q)=%v; want %v, diff %v", test.target, got, test.want, diff)
```

Comment:
Drive-by comment: please don't flip the args. Most (almost all) test diffs are -got +want, flipping will only make it inconsistent / harder to read.

(Nit: please add a line break before the diff.)